src/features/list-view/ListView.js

```diff
@@ -60,7 +60,7 @@ class ListView extends React.PureComponent<Props> {
         const cellData = getGridCell({ columnIndex, rowIndex: rowIndex - 1 });
 
         return (
-            <div className="bdl-ListView-columnCell" key={key} style={style}>
+            <div className={`${CLASS_NAME}-columnCell`} key={key} style={style}>
```

Comment:
Doing this makes it harder to later find-and-replace usages of the CSS classes - is this critical for modularity?

Reply:
Naw, definitely not critical. Just saw it be re-used.